docs/sources/explore/correlations-editor-in-explore.md

```diff
@@ -0,0 +1,118 @@
+---
+labels:
+  products:
+    - enterprise
+    - oss
+title: Correlations Editor in Explore
+weight: 400
+---
+
+# Correlations Editor in Explore
+
+{{% admonition type="note" %}}
+The Explore editor is available in 10.1 and later versions. In the editor, transformations is available in Grafana 10.3 and later versions.
+{{% /admonition %}}
+
+Correlations allow users to build a link between any two datasources. For more information about correlations in general, please see the [documentation]({{< relref "../administration/correlations" >}}) about building correlations in the administration page.
+
+## Create a correlation
+
+1. Go to Explore page.
+1. Select a data source that you would like to be [the source data source]({{< relref "../administration/correlations/correlation-configuration#source-data-source-and-result-field" >}}) for a new correlation.
+1. Run a query producing data in [a supported visualization]({{< relref "../administration/correlations#correlations" >}}).
+1. Click "+ Add" button in the top toolbar and select "Add correlation" (you can also select "Correlations Editor" from the [Command Palette]({{< relref "../search#command-palette" >}})).
+1. Explore is now in Correlations Editor mode indicated by a blue border and top bar. You can exit Correlations Editor using the Exit button in the top bar.
+1. The visualization is enriched with links allowing to create new correlations. Links are attached to the data that you can use to build a new query:
+   - Logs: links are displayed next to field values inside log details for each log row
+   - Table: every table cell is a link
+1. Click on a link to begin adding new correlation. Links are associated with a field that will be used as [a results field of a correlation]({{< relref "../administration/correlations/correlation-configuration" >}}). This means the correlation link will be created on the link you select once the correlation is saved.
+1. Explore opens a split view. Use the right pane to setup [the target query source of the correlation]({{< relref "../administration/correlations/correlation-configuration#target-query" >}}).
+1. Build target query using [variables syntax]({{< relref "../dashboards/variables/variable-syntax" >}}) with variables from the list provided at the top of the pane. The list contains sample values from the selected data row.
+1. Provide optional label and description. Label is used as the name of the link inside the visualization. It can contain variables.
```

Comment:
I think we could add an optional step to indicate transformations can be added at this stage: "Provide optional transformations (see below for more details)."